setup.py

```diff
@@ -5,16 +5,14 @@
     version='0.6.1',
     license='Apache',
     url='https://github.com/obsrvbl/gcp-flowlogs-reader',
-
     description='Reader for Google Cloud VPC Flow Logs',
     long_description=(
         "This project provides a convenient interface for accessing "
         "VPC Flow Logs stored in Google Cloud's Stackdriver Logging service."
     ),
-
+    long_description_content_type='text/x-rst',
```

Comment:
What's this for? I've seen it used when `long_description` is the result of calling `open(some_reST_file).read()` but don't know what it does for plain text.

Reply:
It's to placate up `twine check`, mainly. If we ever put any interesting markup in the `long_description` then it would render it nicely on PyPI.